app/controllers/email_actions_controller.rb

```diff
@@ -19,19 +19,13 @@ def mark_discussion_as_read
   end
 
   def mark_summary_email_as_read
-    @inbox = Inbox.new(user)
-    @inbox.load
-    time_start = Time.at(params[:time_start].to_i).utc
+    time_start  = Time.at(params[:time_start].to_i).utc
     time_finish = Time.at(params[:time_finish].to_i).utc
 
-    Queries::VisibleDiscussions.new(user: user,
-                                    groups: user.inbox_groups).
+    Queries::VisibleDiscussions.new(user: user).
```

Comment:
As a matter of process I think you should avoid rewriting stuff you don't have to. Leaving those bits as small changes to make in the future seems less error prone when doing something this large.